test/conversion_helper-test.js

```diff
@@ -642,7 +642,7 @@ describe('conversion_helper', () => {
 
       context('greater than or equal to 1 light-second', () => {
         it('should convert to light-seconds', () => {
-            verifyConversion(372565, " miles", 2.000001105297986, " light-seconds");
+            verifyConversion(3725650, " miles", 1.8628250000000002, " light-seconds");
```

Comment:
This seems a bit strange, how is it possible that when we increase the miles, the light seconds go down? And Travis is passing?? Could this be an integer overflow issue? Google says the answer should be [20 light seconds](https://www.google.hr/search?q=3725650+miles+to+light+seconds&oq=3725650+miles+to+light+seconds&aqs=chrome..69i57j6.4071j0j4&sourceid=chrome&ie=UTF-8)

Reply:
Because I'm the biggest idiot on the planet and when I changed the threshold for conversion to 2 million miles I also changed the meter to light-second factor. Embarassing really.

Reply:
Ah, no worries, I didn't catch that either, LMAO. Yay for tests?

Reply:
Well at least you noticed it was off and it didn't make sense. As soon as I see green tests I'm happy. Sorry!